.github/workflows/changelog-flow.yml

```diff
@@ -0,0 +1,34 @@
+name: Generate Changelog
+on:
+  release:
+    types: [published]
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          fetch-depth: 0
+          token: ${{ secrets.GH_PAT }}
```

Comment:
Not sure if we have this, but we would have to wait for the maintainer to check this.

Reply:
https://github.com/getporter/porter/blob/main/.github/workflows/close-issues.yaml ig they have used secrets in other workflows too!
Nevertheless, will wait for maintainers to have  a say on this!